e2e/tests/visual-output/rendering/testcases/svg-import.ts

```diff
@@ -0,0 +1,105 @@
+import type { renderTestType } from '../../../types';
+
+/**
+ * Tests for SVG import functionality
+ * These tests load SVG files and compare their rendering to golden images
+ */
+
+const createTestFromSVG = (svgName: string): renderTestType => {
+  if (!svgName) {
+    throw new Error('SVG name is required');
+  }
+
+  return {
+    title: `Svg import test ${svgName}`,
+    golden: `${svgName}.png`,
+    percentage: 0.055,
+    snapshotSuffix: 'svg-import',
+    size: [100, 100],
+    only: true,
+    async renderFunction(canvas, fabric) {
+      const asset = await globalThis.getAsset(`${svgName}.svg`);
+      const { objects, options } = await fabric.loadSVGFromString(asset);
+      const nonNullObj = objects.filter((obj) => !!obj);
+      const group = fabric.util.groupSVGElements(nonNullObj, options);
+      const dims = group._getTransformedDimensions();
+      canvas.setDimensions({
+        width: dims.x + group.left,
+        height: dims.y + group.top,
+      });
```

Comment:
the canvas gets definetely resized to what the content is, so yes we should regenerate the originals

Reply:
alright I will push recreated goldens

Reply:
I checked svg_radial_13.svg asset
it has this
<rect x="0" y="0" width="100" height="100"
    style="fill: url(#focal_set); stroke: black;"/>
</svg>

so width is 100 and height is 100
and playwright output is also those dimensions
however expected one has 300 x 150
so seems like playwright is more precise here anyway

Reply:
they are all transparent now
as I didn't want to regenerate all of them
just the failing ones so I set the bg transparent in the test case